### DISCUSSION TITLE
AVD builds invalid Config

### DISCUSSION THREAD

**Discussion by lukaswolbeck** [aristanetworks/avd#3609] [Q&A]
Hey,
Hello everyone,
I am currently building a config which is similar to the dual DC example of the AVD framework.
The Leafs are all model 7050SX3-48C8.
If I now build the config, I get in the part 'config router bgp <as_number>'--> vlan vlan a configline which is not compatible with the platform. This would be 'rd evpn domain remote <***.***.***.***:***:***>' (asterisks are placeholders).
In my playbook I have already tried 7050X3, 7050SX3 as platform in the DC1 and DC2 groupvars.
Am I missing something?
I need help.
Thanks in advance and best regards

Translated with DeepL.com (free version)

**Comment by carl-baillargeon** [comment]
Hello,

Thank you for your interest in AVD!

In the Dual DC example, the EVPN gateway feature is used to interconnect EVPN VXLAN domains. This feature is only available on Arista R hardware platforms:

- DCS-7020R
- DCS-7280R/R2/R3
- DCS-7500R/R2/R3

You can refer to the following documentation for more details on the feature: https://www.arista.com/en/support/toi/eos-4-26-1f/14785-multi-domain-evpn-vxlan.

The feature is also available in a virtual environment, so you can build a virtual environment with vEOS-lab or cEOS-lab to test this feature or test AVD in general.

If you need help restructuring your AVD project to support your design and platforms, please feel free to reach out!

Thank you.
